back/app/finders/events_finder.rb

```diff
@@ -1,7 +1,7 @@
-# frozen_string_literal: true
+# frozen_string_literal: false
```

Comment:
Is this an automatic change from Rubocop? Looks off to me, I've only ever seen that to be set to `true` (or the magic comment not being there at all).

Reply:
Good catch. I assume this was automatic by Rubocop, as it certainly wasn't me. @koendehondt do you know why this happened? Is it correct or should I revert this?

Reply:
So yes, take it out.

Reply:
Ok, weird. Thanks, I'll revert it.

Reply:
I believe we did it because otherwise `scope.to_s.delete_prefix!('-')` raises `can't modify frozen String: "qwe"`. Though as we removed `scope.to_s.delete_prefix!('-')`, it doesn't matter now.

Reply:
Ahh, thanks Alex. I missed that 👍